/root/miniconda3/envs/idm/lib/python3.10/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
{'variance_type', 'clip_sample_range', 'thresholding', 'dynamic_thresholding_ratio'} was not found in config. Values will be initialized to default values.
/root/miniconda3/envs/idm/lib/python3.10/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
The config attributes {'decay': 0.9999, 'inv_gamma': 1.0, 'min_decay': 0.0, 'optimization_step': 37000, 'power': 0.6666666666666666, 'update_after_step': 0, 'use_ema_warmup': False} were passed to UNet2DConditionModel, but are not expected and will be ignored. Please verify your config.json configuration file.
{'reverse_transformer_layers_per_block', 'dropout'} was not found in config. Values will be initialized to default values.
{'reverse_transformer_layers_per_block', 'dropout', 'addition_embed_type', 'attention_type'} was not found in config. Values will be initialized to default values.
Some weights of the model checkpoint were not used when initializing UNet2DConditionModel:
 ['add_embedding.linear_1.bias, add_embedding.linear_1.weight, add_embedding.linear_2.bias, add_embedding.linear_2.weight']
{'image_encoder'} was not found in config. Values will be initialized to default values.
Loading pipeline components...: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 8/8 [00:00<00:00, 120.10it/s]






















100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:42<00:00,  1.43s/it]
Global_step:  0
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!
Images generated!






















100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:43<00:00,  1.45s/it]






















100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:43<00:00,  1.45s/it]






















100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 30/30 [00:43<00:00,  1.46s/it]











 50%|██████████████████████████████████████████████████████████████████████████████████████████                                                                                          | 15/30 [00:23<00:23,  1.56s/it]
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x7faa1f51d360>
Traceback (most recent call last):
  File "/root/miniconda3/envs/idm/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 1478, in __del__
    self._shutdown_workers()
  File "/root/miniconda3/envs/idm/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 1442, in _shutdown_workers
    w.join(timeout=_utils.MP_STATUS_CHECK_INTERVAL)
  File "/root/miniconda3/envs/idm/lib/python3.10/multiprocessing/process.py", line 149, in join
    res = self._popen.wait(timeout)
  File "/root/miniconda3/envs/idm/lib/python3.10/multiprocessing/popen_fork.py", line 40, in wait
    if not wait([self.sentinel], timeout):
  File "/root/miniconda3/envs/idm/lib/python3.10/multiprocessing/connection.py", line 936, in wait
    ready = selector.select(timeout)
  File "/root/miniconda3/envs/idm/lib/python3.10/selectors.py", line 416, in select
    fd_event_list = self._selector.poll(timeout)
KeyboardInterrupt:
Traceback (most recent call last):
  File "/notebooks/ayna/working_repo/IDM-VTON/train.py", line 351, in <module>
    main()
  File "/notebooks/ayna/working_repo/IDM-VTON/train.py", line 348, in main
    train(args, train_dataloader, model, unet, image_encoder, optimizer, accelerator)
  File "/notebooks/ayna/working_repo/IDM-VTON/train.py", line 203, in train
    images = model(
  File "/root/miniconda3/envs/idm/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/notebooks/ayna/working_repo/IDM-VTON/src/tryon_pipeline.py", line 1823, in __call__
    latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs, return_dict=False)[0]
  File "/root/miniconda3/envs/idm/lib/python3.10/site-packages/diffusers/schedulers/scheduling_ddpm.py", line 441, in step
    beta_prod_t = 1 - alpha_prod_t
  File "/root/miniconda3/envs/idm/lib/python3.10/site-packages/torch/_tensor.py", line 38, in wrapped
    if has_torch_function(args):
KeyboardInterrupt